modules/apps/expando/expando-web/src/main/resources/META-INF/resources/edit/expando.jsp

```diff
@@ -89,14 +89,6 @@ viewAttributesURL.setParameter("modelResource", modelResource);
 
 PortalUtil.addPortletBreadcrumbEntry(request, LanguageUtil.get(request, "view-attributes"), viewAttributesURL.toString());
 
-PortletURL newCustomFieldURL = renderResponse.createRenderURL();
-
-newCustomFieldURL.setParameter("mvcPath", "/edit/select_field_type.jsp");
-newCustomFieldURL.setParameter("redirect", redirect);
-newCustomFieldURL.setParameter("modelResource", modelResource);
-
-PortalUtil.addPortletBreadcrumbEntry(request, LanguageUtil.get(request, "new-custom-field"), newCustomFieldURL.toString());
-
```

Comment:
By removing this, we lose the ability to go back to the select field type page when trying to add a new custom field. Is this intentional?